src/openeo_grass_gis_driver/actinia_processing/load_collection_process.py

```diff
@@ -245,13 +245,45 @@ def create_process_chain_entry(input_object: DataObject,
 
     if input_object.is_raster():
         importer = {"id": "r_info_%i" % rn, "module": "r.info", "inputs": [
-            {"param": "map", "value": input_object.grass_name()}, ], "flags": "g"}
+            {"param": "map", "value": input_object.grass_name()}, ],
+            "flags": "g"}
+
     elif input_object.is_vector():
         importer = {"id": "v_info_%i" % rn, "module": "v.info", "inputs": [
-            {"param": "map", "value": input_object.grass_name()}, ], "flags": "g"}
+            {"param": "map", "value": input_object.grass_name()}, ],
+            "flags": "g"}
+
     elif input_object.is_strds():
         importer = {"id": "t_info_%i" % rn, "module": "t.info", "inputs": [
-            {"param": "input", "value": input_object.grass_name()}, ], "flags": "g"}
+            {"param": "input", "value": input_object.grass_name()}, ],
+            "flags": "g"}
+
+    elif input_object.is_rastercube():
+        instance_id = input_object.instance
```

Comment:
e.g, here the instance_id could be set to mapset. Then the general model for all types wouldn't need to be adjusted and parameters for it are reduced.

Reply:
That was my first approached, but as the `mapset` is required to create the `endpoint` where the process is gonna run. The processes were sent to `/api/v3/<instance_id>/processing_async_export`. This is the way I manage to create the prototype, but I think it would be necessary to define well how to manage `STAC` as a `DataObject` in GRASS. I think the key here is to find a way around to send the `PC` to create the `strds` and then run the process

Reply:
But that `/api/v3/<instance_id>/processing_async_export` wouldn't be a problem, when instance is always named after mapset. But ok - does that currently mean that the STAC to be imported must have the instance named after an existing mapset? In which processing will take place?

Reply:
OK - But how do I define the mapset where the strds is gonna be stored? I think, the problem I encouraged was that.

Reply:
resolved it verbally - the process is send to `/api/v3/<location_name>/processing_async_export` and mapset variable is not used here, so instance_id can be set to mapset and is not needed in the data model.